include/deal.II/grid/tria.h

```diff
@@ -1010,29 +1011,8 @@ namespace internal
  *
  * The Triangulation class has a variety of signals that indicate different
  * actions by which the triangulation can modify itself and potentially
- * require follow-up action elsewhere: - creation: This signal is triggered
- * whenever the Triangulation::create_triangulation or
- * Triangulation::copy_triangulation is called. This signal is also triggered
- * when loading a triangulation from an archive via Triangulation::load. -
- * pre-refinement: This signal is triggered at the beginning of execution of
- * the Triangulation::execute_coarsening_and_refinement function (which is
- * itself called by other functions such as Triangulation::refine_global). At
- * the time this signal is triggered, the triangulation is still unchanged. -
- * post-refinement: This signal is triggered at the end of execution of the
- * Triangulation::execute_coarsening_and_refinement function when the
- * triangulation has reached its final state - copy: This signal is triggered
- * whenever the triangulation owning the signal is copied by another
- * triangulation using Triangulation::copy_triangulation (i.e. it is triggered
- * on the <i>old</i> triangulation, but the new one is passed as a argument).
- * - clear: This signal is triggered whenever the Triangulation::clear
- * function is called. This signal is also triggered when loading a
- * triangulation from an archive via Triangulation::load as the previous
- * content of the triangulation is first destroyed. - any_change: This is a
- * catch-all signal that is triggered whenever the create, post_refinement, or
- * clear signals are triggered. In effect, it can be used to indicate to an
- * object connected to the signal that the triangulation has been changed,
- * whatever the exact cause of the change.
- *
+ * require follow-up action elsewhere. Please refer to
+ * @ref Triangulation::Signals for details.
```

Comment:
Actually, `@ref` is wrong here because you're not referring to a label. You don't need any `@`-command at all, doxygen gets it right automatically.